index.html

```diff
@@ -1,12 +1,75 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
-</head>
-<body>
-    <h1>Hello world</h1>
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;600;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header class="header">
+      <div class="logo-container"><p class="my-logo">Deborah Fashoro</p></div>
+      <div class="menu-container">
+        <img src="./images/Icon-Menu.png" alt="menu-hamburger-icon" />
+      </div>
+      <nav>
+        <a href="#">Portfolio</a>
+        <a href="#">About</a>
+        <a href="#">Contact</a>
+      </nav>
+    </header>
+    <main>
+      <!-- Headline Section-->
+      <section class="headline">
+        <div class="header-bg">
+          <div class="about-me-text-container">
+            <div class="primary-text">
+              <p>I'm Deborah.</p><p>Glad to see you!</p>
+            </div>
+            <p class="secondary-text">
+              I’m a software developer! I can help you build a product , feature
+              or website Look through some of my work and experience! If you
+              like what you see and have a project you need coded, don’t
+              hesitate to contact me.
+            </p>
+            <p class="supporting-text">LET'S CONNECT</p>
+          </div>
+          <div class="social-icons">
+            <div>
+              <a href="#"
+                ><img src="./images/twitter.png" alt="twitter-icon"
+              /></a>
+            </div>
+            <div>
+              <a href="#"
+                ><img src="./images/linkedin.png" alt="linkedin-icon"
+              /></a>
+            </div>
+            <div>
+              <a href="#"
+                ><img src="./images/vector-1.png" alt="social-handle-icon"
+              /></a>
+            </div>
+            <div>
+              <a href="#"
+                ><img src="./images/vector-2.png" alt="social-handle-icon"
+              /></a>
+            </div>
+            <div>
+              <a href="#"
+                ><img src="./images/github.png" alt="github-icon"
+              /></a>
+            </div>
+          </div>
+        </div>
```

Comment:
- [ ] Please consider using `ul` for the whole list and `li` for list item. `<div class="social-icons">` should be a `ul` and each of the `div` inside (direct children) it should be `li`. This will make your project way more sementic and which will improve the user experience 😄 😸

Reply:
Fixed